runtime/src/main/resources/META-INF/quarkus-extension.yaml

```
@@ -5,7 +5,7 @@ metadata:
      - "hibernate"
      - "panache"
      - "jpa"
-   guide: https://quarkiverse.github.io/quarkiverse-docs/jpastreamer/dev/
+   guide: https://github.com/quarkiverse/quarkus-jpastreamer#readme
```

I saw that a documentation existed so I integrated it into the quarkiverse-docs repository. It is now published to:

```suggestion
   guide: https://quarkiverse.github.io/quarkiverse-docs/quarkus-jpastreamer/dev/
```

Note that it still needs fixing as the current guide URL misses the `quarkus-` prefix.

@julgus I had to push this commit https://github.com/quarkiverse/quarkus-jpastreamer/commit/ac3e3e2f517126223357f3ccfe0a78a74dff0e02 to the repository as we are not really supporting multi versions right now and we need a stable URL. We should probably improve that in the future.

---

Even better - thanks, @gsmet !